paasta_tools/tron_tools.py

```diff
@@ -327,15 +327,31 @@ def format_tron_job_dict(job_config, cluster_fqdn_format, default_paasta_cluster
     # Only pass non-None values, so Tron will use defaults for others
     return {key: val for key, val in result.items() if val is not None}
 
+def _update_config(config, tron_conf_path):
+    tron_config_file = service_configuration_lib._read_yaml_file(tron_conf_path)
```

Comment:
I used `_read_yaml_file` as a hack because the filepath was different. But for the service directory, I think we should use `read_extra_service_information` like Chronos/Marathon do: https://github.com/Yelp/paasta/blob/master/paasta_tools/chronos_tools.py#L198

Reply:
Good call. qui++